### DISCUSSION TITLE
How to enter the full screen while being in presentation mode?

### DISCUSSION THREAD

**Discussion by Kapelianovych** [motion-canvas/motion-canvas#837] [Q&A] [answered]
Hello.
I have read the [Presentation](https://motioncanvas.io/docs/presentation) section where it is mentioned that one can switch to full screen, so all controls disappears. Unfortunately there is not a word about how to do that and in the online editor I can only reach a following state. I tried to search for a button that enters full screen or something like that but failed.
<img width="1324" alt="Screenshot 2023-11-05 at 15 48 31" src="https://github.com/motion-canvas/motion-canvas/assets/28602579/f5ed5928-fb7d-4f04-91af-6c585eeaa2bb">

**Comment by milotheirself** [comment, marked as answer]
The presentation view is a drop-down option where you would normally start the render, from there you can switch your browser to full screen by pressing `F11`. If you don't see the drop-down button, check if you're maybe using an older version that didn't have it.

![image](https://github.com/motion-canvas/motion-canvas/assets/54866561/3bc3ec79-62d1-4d53-8789-28fb9dbb32fa)
